### PR TITLE
Revamp documentation to better describe current AoD status

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

AoD has several changes that are not properly reflected in the documentation
- EC3 should be featured as top-level instead of cloud compute (it was not even in AoD!)
- Exoscale vouchers are not really active
- Chipster is not really now managed by FCSG

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
@thebe14 can you take a look at this one? It may conflict a bit with some of the work you are doing and I should adapt to that :)

**Comment by enolfc** [comment]
> The FSGS is also down/no more working an admins are not answering, with @glarocca we agreed to remove it, so it would be good to also remove the related doc.

This is now gone (at least `git grep -i fsgs` does not show up anything)

File: content/en/users/getting-started/_index.md [hunk 1]
```diff
@@ -130,6 +130,80 @@ When EGI is able to support a request for resources, it can do so in two ways:
      Agreement (SLA) with you
    - A new VO will be created for your community
 
+## Pilot your application
+
+EGI offers a plaground allocation for users get access to the services and
+understand how to port applications and develop new data analytics tools that
+can be turn into online services that can be accessed by scientist worldwide.
```
Comment (gwarf) [inline content/en/users/getting-started/_index.md:137]:
```suggestion
EGI offers a playground allocation for users to get access to the services and
understand how to port applications and develop new data analytics tools that
can be turn into online services that can be accessed by scientist worldwide.
```

File: content/en/users/getting-started/_index.md [hunk 2]
```diff
@@ -130,6 +130,80 @@ When EGI is able to support a request for resources, it can do so in two ways:
...
+### Requirements and user registration
+
+Access **requires** acceptance of
+[Acceptable Use Policy (AUP) and Conditions of the \'EGI Applications on Demand Service\'](https://documents.egi.eu/public/ShowDocument?docid=2635).
+
+{{% alert title="Acknowledgment" color="info" %}}
+
+Users of the service are asked to provide appropriate acknowledgement of the use
+in scientific publications. The following acknowledgement text can be used for
+this purpose (you can adapt to match the exact providers in your case):
```
Comment (gwarf) [inline content/en/users/getting-started/_index.md:148]:
```suggestion
this purpose (you should adapt to match the exact providers in your case):
```

File: content/en/users/getting-started/_index.md [hunk 3]
```diff
@@ -130,6 +130,80 @@ When EGI is able to support a request for resources, it can do so in two ways:
...
+
+**This work used advanced computing resources from the 100%IT, CESGA, CLOUDIFIN,
+CYFRONET-CLOUD, GSI-LCG2, IFCA-LCG2, IN2P3-IRES, INFN-CATANIA-STACK,
+INFN-PADOVA-STACK, SCAI, TR-FC1-ULAKBIM, UA-BITP and UNIV-LILLE resource centres
+of the EGI federation. The services are co-funded by the EGI-ACE project (grant
+number 101017567).** {{% /alert %}}
+
+When requesting access users are guided through a registration process. Members
+of the EGI support team will perform a lightweight vetting process to validate
+the users\' requests before granting the access to the resources.
```
Comment (gwarf) [inline content/en/users/getting-started/_index.md:158]:
```suggestion
the users' requests before granting the access to the resources.
```
Should not be needed, right?

File: content/en/users/getting-started/_index.md [hunk 4]
```diff
@@ -130,6 +130,80 @@ When EGI is able to support a request for resources, it can do so in two ways:
...
+
+### Get access to pilot allocation
+
+1. Login to the [EGI Marketplace](https://marketplace.egi.eu) with the
```
Comment (gwarf) [inline content/en/users/getting-started/_index.md:162]:
```suggestion
1. Log into the [EGI Marketplace](https://marketplace.egi.eu) with the
```

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, LGTM, but added some small fixes to be included before we merge.

**Review by gwarf — APPROVED** [review]

**Pull request merged**